racket/collects/racket/match/parse.rkt

```diff
@@ -52,6 +53,16 @@
           (rest suffix)
           (if (eq? ddk-size #t) 0 ddk-size)))
 
+(define (make-pregexp-matcher r)
+  #`(lambda (e)
```

Comment:
It seems like this function could be defined once and then referenced in the generated code.

Reply:
Fixed.